src/rpc/cli.cpp

```diff
@@ -160,20 +160,20 @@ char* my_generator(const char* text, int state)
 }
 
 
+#ifdef HAVE_READLINE
```

Comment:
This changes seems not correct, as it removed definition/implementation of a function, need to make sure that it's not used somewhere else.

Reply:
This fixes the unused function warning when compiled without readline. There is but a single call of the function protected by another HAVE_READLINE. See cli::getline@cli.cpp

Reply:
Good to know. Thanks.